candi/cloud-providers/aws/openapi/cluster_configuration.yaml

```diff
@@ -3,6 +3,7 @@ apiVersions:
 - apiVersion: deckhouse.io/v1
   openAPISpec:
     type: object
+    default: {}
```

Comment:
Why we need this default ?

Reply:
These defaults are not needed, I forgot to remove them after the tests.